pages/persoonsgegevens/[person].tsx

```diff
@@ -233,13 +239,27 @@ export default function MultistepForm1() {
                   </FormField>
                   <DeclarationCheckboxGroup register={register} checkboxData={checkboxData} />
                   <Button
-                    disabled={!formState.isValid || loading}
                     type="submit"
                     name="type"
                     appearance="primary-action-button"
+                    aria-describedby={invalidStateDescriptionId}
+                    busy={loading}
                   >
                     Contactgegevens opslaan
                   </Button>
+                  {!formState.isValid && (
+                    <>
+                      {!formState.isSubmitted ? (
+                        <Paragraph hidden id={invalidStateDescriptionId}>
+                          Vul eerst alle gegevens in.
+                        </Paragraph>
```

Comment:
Deze paragraph kan ook wel in de `FormFieldDescription`, for consistency. Dan moet de `hidden` misschien daar naartoe verhuizen.